2025-02-06 22:10:42,645 INFO    Thread-34 :662016 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2025-02-06 22:10:42,645 INFO    Thread-34 :662016 [wandb_setup.py:_flush():79] Configure stats pid to 662016
2025-02-06 22:10:42,646 INFO    Thread-34 :662016 [wandb_setup.py:_flush():79] Loading settings from C:\Users\admin\.config\wandb\settings
2025-02-06 22:10:42,646 INFO    Thread-34 :662016 [wandb_setup.py:_flush():79] Loading settings from C:\Users\admin\Documents\GitHub\pum-24\wandb\settings
2025-02-06 22:10:42,646 INFO    Thread-34 :662016 [wandb_setup.py:_flush():79] Loading settings from environment variables: {'project': 'mnist-classifier', 'entity': 'tiopirfur-uniwersytet-jagiello-ski-w-krakowie', 'root_dir': 'C:\\Users\\admin\\Documents\\GitHub\\pum-24', 'sweep_id': 'b3usr0s1', 'run_id': 'ftnitlwa', 'sweep_param_path': 'C:\\Users\\admin\\Documents\\GitHub\\pum-24\\wandb\\sweep-b3usr0s1\\config-ftnitlwa.yaml'}
2025-02-06 22:10:42,647 INFO    Thread-34 :662016 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2025-02-06 22:10:42,647 INFO    Thread-34 :662016 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2025-02-06 22:10:42,647 INFO    Thread-34 :662016 [wandb_setup.py:_flush():79] Applying login settings: {}
2025-02-06 22:10:42,647 INFO    Thread-34 :662016 [wandb_init.py:_log_setup():533] Logging user logs to C:\Users\admin\Documents\GitHub\pum-24\wandb\run-20250206_221042-ftnitlwa\logs\debug.log
2025-02-06 22:10:42,648 INFO    Thread-34 :662016 [wandb_init.py:_log_setup():534] Logging internal logs to C:\Users\admin\Documents\GitHub\pum-24\wandb\run-20250206_221042-ftnitlwa\logs\debug-internal.log
2025-02-06 22:10:42,648 INFO    Thread-34 :662016 [wandb_init.py:_jupyter_setup():479] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001F903F63940>
2025-02-06 22:10:42,648 INFO    Thread-34 :662016 [wandb_init.py:init():619] calling init triggers
2025-02-06 22:10:42,648 INFO    Thread-34 :662016 [wandb_init.py:init():626] wandb.init called with sweep_config: {'batch_size': 32, 'layer_1_size': 64, 'layer_2_size': 64, 'learning_rate': 0.001}
config: {}
2025-02-06 22:10:42,649 INFO    Thread-34 :662016 [wandb_init.py:init():669] starting backend
2025-02-06 22:10:42,649 INFO    Thread-34 :662016 [wandb_init.py:init():673] sending inform_init request
2025-02-06 22:10:42,675 INFO    Thread-34 :662016 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn, using: spawn
2025-02-06 22:10:42,684 INFO    Thread-34 :662016 [wandb_init.py:init():686] backend started and connected
2025-02-06 22:10:42,768 INFO    Thread-34 :662016 [wandb_run.py:_config_callback():1387] config_cb None None {'batch_size': 32, 'layer_1_size': 64, 'layer_2_size': 64, 'learning_rate': 0.001}
2025-02-06 22:10:42,770 INFO    Thread-34 :662016 [wandb_run.py:_label_probe_notebook():1339] probe notebook
2025-02-06 22:10:42,777 INFO    Thread-34 :662016 [wandb_init.py:init():781] updated telemetry
2025-02-06 22:10:42,778 INFO    Thread-34 :662016 [wandb_init.py:init():814] communicating run to backend with 90.0 second timeout
2025-02-06 22:10:43,498 INFO    Thread-34 :662016 [wandb_init.py:init():867] starting run threads in backend
2025-02-06 22:10:44,793 INFO    Thread-34 :662016 [wandb_run.py:_console_start():2456] atexit reg
2025-02-06 22:10:44,793 INFO    Thread-34 :662016 [wandb_run.py:_redirect():2305] redirect: wrap_raw
2025-02-06 22:10:44,793 INFO    Thread-34 :662016 [wandb_run.py:_redirect():2370] Wrapping output streams.
2025-02-06 22:10:44,793 INFO    Thread-34 :662016 [wandb_run.py:_redirect():2395] Redirects installed.
2025-02-06 22:10:44,796 INFO    Thread-34 :662016 [wandb_init.py:init():911] run started, returning control to user process
2025-02-06 22:10:44,796 INFO    Thread-34 :662016 [wandb_run.py:_finish():2155] finishing run tiopirfur-uniwersytet-jagiello-ski-w-krakowie/mnist-classifier/ftnitlwa
2025-02-06 22:10:44,797 INFO    Thread-34 :662016 [jupyter.py:save_history():459] not saving jupyter history
2025-02-06 22:10:44,797 INFO    Thread-34 :662016 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2025-02-06 22:10:44,797 INFO    Thread-34 :662016 [wandb_init.py:_jupyter_teardown():461] cleaning up jupyter logic
2025-02-06 22:10:44,797 INFO    Thread-34 :662016 [wandb_run.py:_atexit_cleanup():2420] got exitcode: 1
2025-02-06 22:10:44,797 INFO    Thread-34 :662016 [wandb_run.py:_restore():2402] restore
2025-02-06 22:10:44,797 INFO    Thread-34 :662016 [wandb_run.py:_restore():2408] restore done
2025-02-06 22:10:47,394 INFO    Thread-34 :662016 [wandb_run.py:_footer_history_summary_info():3960] rendering history
2025-02-06 22:10:47,394 INFO    Thread-34 :662016 [wandb_run.py:_footer_history_summary_info():3992] rendering summary
2025-02-06 22:10:47,413 INFO    Thread-34 :662016 [wandb_run.py:_footer_sync_info():3921] logging synced files
